microprofile/WFLY-14940_upgrade-microprofile-health-3.1.adoc

```diff
@@ -0,0 +1,69 @@
+= Upgrade MicroProfile Health to 3.1
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 4.1 provides a minor upgrade for https://github.com/eclipse/microprofile-health/releases/tag/3.1[MicroProfile Health 3.1] with changes described in its https://download.eclipse.org/microprofile/microprofile-health-3.1/microprofile-health-spec-3.1.html#release_notes_3_1[Release Notes].
+
+This component upgrade also covers the upgrade of smallrye-health 3.1.1 that implements MicroProfile Health 3.1.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-14940[WFLY-14040]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1763[EAP7-1763]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+[X] Engineering
+[ ] QE
+
+== Requirements
+
```

Comment:
Actually to me this list seems more like an implementation plan for the WildFly to be updated as runtime in order to support MicroProfile Health 3.1. specs, rather than requirements.

Requirements - at least the functional ones - should define how the specific WidFly runtime will have to behave.
If this is defined by the specs already, then just a link to the document would be enough.
WDYT?

Reply:
This was taken from different AD (RM) and I think it is relevant as all of these are requirements but I will add also functional ones.

Reply:
Can be resolved (I don't have the grants for doing so), thanks.